atom/index.js

```diff
@@ -21,6 +21,9 @@ export let atom = (initialValue, level) => {
       }
       return store.value
     },
+    peek() {
+      return store.value
```

Comment:
`store.value` is not always `Value`. Is store has no subscribers, the `store.value` will be `undefined` 

Here is an example
https://github.com/nanostores/nanostores/blob/main/computed/index.js#L20

I am thinking of moving `store.value` to a public API with `Value | undefined` type. Will it work?